core/src/main/java/io/confluent/rest/Application.java

```diff
@@ -370,6 +386,8 @@ public Handler configureWebSocketHandler() throws ServletException {
   }
 
   // This is copied from the old MAP implementation from cp ConfigDef.Type.MAP
+  // TODO: This method should be removed in favor of RestConfig.getMap(). Note
+  // that it is no longer used by projects related to kafka-rest.
```

Comment:
Note: I plan to make changes to Schema Registry following this set of PRs being merged to clean up Application.